addons/project/models/project.py

```diff
@@ -245,6 +246,14 @@ def _compute_alias_enabled(self):
         for project in self:
             project.alias_enabled = project.alias_domain and project.alias_id.alias_name
 
+    @api.depends("alias_name", "alias_domain")
+    def _compute_alias_email_formatted(self):
+        for project in self:
+            before_at_domain = ""
+            if project.alias_name:
+                before_at_domain = project.alias_name.split("@")[0]
+            project.alias_email_formatted = "%s@%s" % (before_at_domain, project.alias_domain)
```

Comment:
Not sure "email formatted" should be "help@" if alias_domain is not set. I would vote to keep False to be coherent with other use of aliases display within Odoo (no alias_domain -> all display of aliases is voided).

Reply:
The alias_email_formatted is only used as a read only field and this field is displayed if and only if there is an alias_domain enabled.

Reply:
Well display is different from actual content. Hiding an incomplete value is good, but I think it is better to not let people having access to an incomplete value (I could do project.alias_email_formatted a bit anywhere, even in a frontend template as a customization if I want, or within an email template).